plat/allwinner/common/sunxi_bl31_setup.c

```diff
@@ -86,6 +129,17 @@ void bl31_platform_setup(void)
 
 	generic_delay_timer_init();
 
+	fdt = sunxi_find_dtb();
+	if (fdt) {
```

Comment:
I think `!= NULL` is expected here

Reply:
That's only to make Coverity happy. If in Allwinner platforms you don't care about it, don't worry about that kind of details.